apps/mymedicare_cb/views.py

```diff
@@ -109,6 +109,11 @@ def callback(request):
         crosswalk.fhir_id = fhir_id
         crosswalk.save()
 
+        logger.info("Beneficiary connected to FHIR: %s" % crosswalk.fhir_id)
```

Comment:
Sorry, on second thought, I don't think I'm comfortable logging this information. User IDs should stay out of our logs.